### PR TITLE
Fix driver naming, fingerprint output, races/tests

### PR DESCRIPTION
**Pull request by schmichael**
Sorry for the random assortment of fixes. These were mostly encountered when working on implementing agent restarting (TR.Restore+Driver.RecoverTask), and I wanted to pull them out into their own PR as the agent restarting PR will be pretty complex on its own.

* registered rkt (on linux)
* made all exported (and many internal) driver package names consistent
* made all fingerprint messages consistent (should we use a const)
* fixed a ubiquitous race in driver handles (driver tests pass with `-race` now!)
* fix a comment on the logger that got copy/pasted around
* squelch error log line when Fingerprint gets canceled

### CODE REVIEW

File: drivers/qemu/handle.go [hunk 1]
```diff
@@ -27,19 +28,24 @@ type qemuTaskHandle struct {
 }
 
 func (h *qemuTaskHandle) IsRunning() bool {
+	h.stateLock.RLock()
+	defer h.stateLock.RUnlock()
 	return h.procState == drivers.TaskStateRunning
 }
 
 func (h *qemuTaskHandle) run() {
```
Comment (jippi) [inline drivers/qemu/handle.go]: `taskHandle`?

File: drivers/rawexec/driver.go [hunk 2]
```diff
@@ -154,25 +157,26 @@ type RawExecTaskState struct {
...
+		tasks:             newTaskStore(),
+		ctx:               ctx,
+		signalShutdown:    cancel,
+		fingerprintPeriod: fingerprintPeriod,
+		logger:            logger,
 	}
 }
 
-func (r *RawExecDriver) PluginInfo() (*base.PluginInfoResponse, error) {
+func (r *Driver) PluginInfo() (*base.PluginInfoResponse, error) {
```
Comment (notnoop) [inline drivers/rawexec/driver.go]: It makes more sense for the receiver now to be `d` too

**Comment by schmichael** [comment]
Ready for re-review

**Comment by schmichael** [comment]
@notnoop Gah! Thanks for catching that! Fixed and merging.

**Pull request merged**